### PR TITLE
Converted games views to HAML

### PR DESCRIPTION
**Pull request by JonasThers**

### CODE REVIEW

File: app/views/games/info.html.haml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+%h1 Info
```
Comment (valscion) [inline app/views/games/info.html.haml:1]: This used to be an `h2` so should we keep it as `h2` or is the `h1` good?

```suggestion
%h2 info
```

File: app/views/games/info.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+%p Database contains #{@games_total} games
+- @genre_counts.each do |key, value|
+    %p #{value} are #{key} games
```
Comment (valscion) [inline app/views/games/info.html.haml:5]: This used to be `<%= key %>s` so do we want to keep the same behavior or use the new one?

```suggestion
    %p #{value} are #{key}s
```

File: app/views/games/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    %h2 Games:
+    - if !@games.empty?
+        = paginate @games
+        - @games.each do |game|
+            = link_to game do
+                %div
+                    %h1
+                    = game.name
+                    %span
+                        (#{game.year})
```
Comment (valscion) [inline app/views/games/index.html.haml:13]: Is there a bug here? I think this now renders an empty `<h1>` tag, then after the closing `</h1>` it will render the game's name without any wrapping element and then after that it will render the `<span>` with the game year inside it.

Previously it was like this:

https://github.com/venuu/video-game-database-ruby/blob/7d68e4f3f73ddf0a02a9dbf28b7bd7d5bb3fa80d/app/views/games/index.html.erb#L10

If we want to keep the same behavior where all of these are rendered inside `h3` then I think this would work:

```suggestion
                    %h3
                        = game.name
                        %span
                            (#{game.year})
```

So basically use the `%h3` as the element name and indent all the text contents inside it.

File: app/views/games/info.html.haml (hunk 2 above)
Comment (JonasThers) [reply app/views/games/info.html.haml:5]: This was just a minor visual change, so instead of saying for example "3 are strategys", it would say "3 are strategy games"

Comment (valscion) [reply app/views/games/info.html.haml:5]: Yeah makes sense :+1:

The reason I was asking is that this functionality change was part of a commit titled "Converted ..." so I assumed that the functionality would not change between the ERB version and the HAML version and thus all the changes that there were, were there by accident.

**Comment by valscion** [comment]
Note that you'll need to remove the `Review: Feedback given` label when you think you've applied all the changes and want me to review the code again ☺️. That is the way we signal that another review round is in order.

**Review by valscion — APPROVED** [review]
:+1:
